.travis.yml

```diff
@@ -2,9 +2,10 @@ language: python
 dist: xenial
 python:
   - 2.7
-  - 3.5
   - 3.6
   - 3.7
+  - 3.8
+  - 3.9
   - pypy
```

Comment:
Wanna quote the Python versions? 3.10 is coming!

Reply:
check-python-versions [will take care of it](https://github.com/mgedmin/check-python-versions/commit/65496640974367f81920a451e997b0d85eda8a13).